src/pybind/kaldi/__init__.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.insert(0, os.path.dirname(__file__))
+
+from kaldi_pybind import *
+
```

Comment:
I suppose there are going to be a lot of decisions about which things to put in the global `kaldi` namespace.  But it's OK.

Reply:
There are lots of classes in `kaldi_pybind` and it is a pain to list each one explicitly.

Reply:
Sure, that's OK; kaldi is just a thin wrapper for kaldi_pybind.  I was actually talking about the lines below, where you import things like Matrix.  But as I said, it's OK.